src/components/Chat/MessageWithLink.js

```diff
@@ -70,14 +70,14 @@ const styles = StyleSheet.create({
     marginTop: 5
   },
   authorName: (isMe) => ({
-    color: isMe ? COLORS.white : COLORS.blackgrey,
+    color: isMe ? COLORS.almostBlack : COLORS.gray400,
     fontFamily: fonts.inter[400],
     fontSize: 12,
     marginBottom: 5,
     lineHeight: 18
   }),
   containerPreview: (isMe) => ({
-    backgroundColor: isMe ? COLORS.tradewind : COLORS.lightgrey,
+    backgroundColor: isMe ? COLORS.tradewind : COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the background color of the container preview from `COLORS.lightgrey` to `COLORS.gray100` when `isMe` is false could also affect visibility and contrast. Please verify that this change is consistent with your design guidelines and maintains good usability.
```diff
-    backgroundColor: isMe ? COLORS.tradewind : COLORS.lightgrey,
+    backgroundColor: isMe ? COLORS.tradewind : COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->